pyresample/test/test_utils.py

```diff
@@ -611,3 +611,43 @@ def test_norotation_yaml(self):
         test_area = load_area(f.name, 'regionB')
         self.assertEqual(test_area.rotation, 0)
         os.remove(f.name)
+
+
+class TestNetcdfCFAreaParser(unittest.TestCase):
+    """ Test loading an area definition from netCDF/CF files """
+
+    def test_load_cf_from_wrong_filepath(self):
+        from pyresample.utils import load_cf_area
+
+        # wrong case #1: the path does not exist
+        cf_file = os.path.join(os.path.dirname(__file__), 'test_files', 'does_not_exist.nc')
+        self.assertRaises(ValueError, load_cf_area, cf_file)
```

Comment:
Python 3 has a `FileNotFoundError`, should that be used instead ?

Reply:
done.